### PR TITLE
(dropbox) issue with helper.ps1 (#1272) [AU dropbox]

### PR DESCRIPTION
**Pull request by RedBaron2**
@AdmiringWorm @majkinetor @gep13 
This will fix Issue #1270 
Due to a Posh v3 variable used that defaults to null in Posh v2 systems. Changed to use same variable if not defined. Fixed the version check to check for pre version suffix and remove it.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [x] Issue #1270 
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/dropbox/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ﻿$ErrorActionPreference  = 'Stop'
+ if(!$PSScriptRoot){ $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent }
 . "$PSScriptRoot\helper.ps1"
 
 $version = '72.3.127-beta'
 
 if (!(IsVersionAlreadyInstalled $version)) {
+    if ( $version -match "beta" ) { $version = $version -replace("-beta","") }
```
Comment (AdmiringWorm) [inline automatic/dropbox/tools/chocolateyinstall.ps1]: Why did you add this line?
It isn't used anywhere else, other than in the version installed check.

File: automatic/dropbox/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -1,9 +1,11 @@
 ﻿$ErrorActionPreference  = 'Stop'
+ if(!$PSScriptRoot){ $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent }
```
Comment (AdmiringWorm) [inline automatic/dropbox/tools/chocolateyinstall.ps1]: :+1:

File: automatic/dropbox/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (RedBaron2) [reply automatic/dropbox/tools/chocolateyinstall.ps1]: The version check is trying to check the version installed which dropbox doesn't install with a `-beta` tag. I can make a gist of what I'm saying because I have the `--pre` version installed. The current check is a `False` and will install the dropbox version again.
Mainly it is a fix that should go in the `helper.ps1` file.

Comment (RedBaron2) [reply automatic/dropbox/tools/chocolateyinstall.ps1]: I'll make the change in the update file so a valid version is used for the check.

Comment (AdmiringWorm) [reply automatic/dropbox/tools/chocolateyinstall.ps1]: If that was the intention, then you added the replacement check in the wrong place, it would have needed to be added before the version check takes place, not after.

Anyways, it is better to just pass the actual version during update than doing such replacements during install.

**Comment by AdmiringWorm** [comment]
I'll force out a new stable release of dropbox with this fix.
